iOS/Core/PixelEvent.swift

```diff
@@ -244,6 +244,16 @@ extension Pixel {
         case daxDialogsEndOfJourneyNewTabUnique
         case daxDialogsEndOfJourneyDismissed
 
+        case onboardingTrySearchDialogNewTabDismissButtonTapped
+        case onboardingSearchResultDialogDismissButtonTapped
+        case onboardingTryVisitSiteDialogNewTabDismissButtonTapped
+        case onboardingTryVisitSiteDialogDismissButtonTapped
+        case onboardingTrackersDialogDismissButtonTapped
+        case onboardingFireDialogDismissButtonTapped
+        case onboardingEndOfJourneyDialogNewTabDismissButtonTapped
+        case onboardingEndOfJourneyDialogDismissButtonTapped
+        case onboardingPrivacyPromoDialogDismissButtonTapped
```

Comment:
One thought here is if we should document these as per [this](https://app.asana.com/0/1200194497630846/1209770134560549/f). You can use best judgement here.